web/static/css/app.css

```diff
@@ -352,3 +352,40 @@ code {
     margin-left: -50vw;
   }
 }
+
+.lm-feedback-label:before {
```

Comment:
I couldn't find much in regards to [`:before`](http://tachyons.io/docs/table-of-styles/) in tachyons.

I could have made it more tachyon-esc though. Will look at doing this in future